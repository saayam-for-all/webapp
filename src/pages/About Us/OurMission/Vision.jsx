export function Vision() {
  return (
    <section className="bg-[#F3FBFF] py-16">
      <div className="max-w-[1100px] mx-auto flex justify-between items-start px-4 gap-2">
        {/* Left Text Block */}
        <div className="w-[42%]">
          <h2 className="text-2xl font-bold mb-4">Our Vision</h2>
          <p className="text-gray-700 text-base leading-relaxed">
            Our vision is to cultivate a community where the ethos of "Manava
            Sevaye Madhava Seva" (Service to mankind is Service to God) and
            "Sarve jana sukhino bhavantu" (May all live happily) guides our
            actions, driving positive change and fostering a more compassionate
            and equitable world for all.
          </p>
        </div>

        {/* Right Image Grid */}
        <div className="grid grid-cols-2 gap-1 w-[50%]">
          <img
            src="https://cdn.builder.io/api/v1/image/assets/TEMP/9e489c91f28508597741986fff385e42dcf07f04"
            alt="Vision image 1"
            className="object-cover h-[150px] w-[70%] ml-[50px]" // ðŸ‘ˆ aligned right
          />
          <img
            src="https://cdn.builder.io/api/v1/image/assets/TEMP/96f2577da3ff33600f091becd5353f37e9ec91fd"
            alt="Vision image 2"
            className="object-cover h-[210px] w-full ml-[-20px]"
          />
          <img
            src="https://cdn.builder.io/api/v1/image/assets/TEMP/d69a9a1b73ece3f214ca700fdad2926ac86ffb47"
            alt="Vision image 3"
            className="object-cover h-[150px] w-[70%] mt-[-50px] ml-[50px]" // ðŸ‘ˆ aligned right
          />
          <img
            src="https://cdn.builder.io/api/v1/image/assets/TEMP/7eed748020e8f83d968948c1754918c0f51ff363"
            alt="Vision image 4"
            className="object-cover h-[100px] w-full ml-[-20px]"
          />
        </div>
      </div>
    </section>
  );
}
