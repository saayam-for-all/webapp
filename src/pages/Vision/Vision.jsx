import React from "react";
import { useTranslation } from "react-i18next";
import "./Vision.css";
import VISIONIMG from "../../assets/vision.png";

const MissionVision = () => {
  const { t } = useTranslation();
  return (
    <div className="lg:px-5 md:px-10 lg:px-20 mt-6 mb-6">
      <div>
        <h1 className="text-3xl font-semibold text-center">{t("VISION")}</h1>
        <div className="flex flex-col lg:flex-row max-w-5xl mx-auto mt-5">
          <p className="text-lg lg:px-5 text-justify">
            {t("VISION_DESCRIPTION")}
          </p>

          <div className="flex-shrink-0 items-center flex justify-center">
            <img
              src={VISIONIMG}
              alt="vision"
              className="w-[30rem] h-80 mx-auto"
            />
          </div>
        </div>
      </div>
    </div>
  );
};

export default MissionVision;
