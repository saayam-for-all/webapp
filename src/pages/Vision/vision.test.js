import { render } from "@testing-library/react";
import Vision from "./Vision";

describe("Vision", () => {
  it("renders correctly", () => {
    const tree = render(<Vision />);
    expect(tree).toMatchSnapshot();
  });
});
