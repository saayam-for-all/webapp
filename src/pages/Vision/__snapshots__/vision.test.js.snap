// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vision renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="px-20 mt-6"
      >
        <div>
          <h1
            class="text-2xl font-semibold text-center mt-6"
          >
            mockTranslate(VISION)
          </h1>
          <div
            class="flex items-start"
          >
            <p
              class="mt-5 text-lg px-5"
            >
              mockTranslate(VISION_DESCRIPTION)
            </p>
            <div>
              <img
                alt="vision"
                class="w-[350rem] h-80 mt-5"
                src="vision.png"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="px-20 mt-6"
    >
      <div>
        <h1
          class="text-2xl font-semibold text-center mt-6"
        >
          mockTranslate(VISION)
        </h1>
        <div
          class="flex items-start"
        >
          <p
            class="mt-5 text-lg px-5"
          >
            mockTranslate(VISION_DESCRIPTION)
          </p>
          <div>
            <img
              alt="vision"
              class="w-[350rem] h-80 mt-5"
              src="vision.png"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
